sentinelsat/sentinel.py

```diff
@@ -91,15 +91,18 @@ def _api2dhus_url(api_url):
     def _req_dhus_stub(self):
         try:
             resp = self.session.get(
-                self.api_url + "api/stub/version", auth=self.session.auth, timeout=self.timeout
+                self._api2dhus_url(self.api_url) + "api/stub/version",
+                auth=self.session.auth,
+                timeout=self.timeout,
```

Comment:
What is the motivation for changing this?

Reply:
This part of the code used to try to get a response from scihub.copernicus.eu/apihub/api/stub/version. This always returned a 404. Only after handling that 404 would the code try to get a response from scihub.copernicus.eu/dhus/api/stub/version. 
I don't know what the motivation for that initial attempt was, but It seems useless to me. That's why now it tries the dhus url from the start. 
Also there was a problem with the handling of the 404 error, something about logger not converting everything to strings that I couldn't quite figure out. So actually the error handler ran into an error and it was all a big mess. 
I was going to look into it more but I wanted to get the CLI working first.

Reply:
The rationale for trying the original URL first would be that users can provide e.g. an API URL to a national DHuS endpoint or so. If that URL for some reason includes the string `apihub` anywhere, it will get replaced, which might lead to a URL that does not even exist. So the original behavior, trying the original URL first, is better. Could you please revert this?

I think the conflicting line about logging is this one:
```python
self.logger.error("HTTPError: ", err)
```

When you change that to
```python
self.logger.error("HTTPError: %s", err)
```
it should work.